tests/test_integration.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import flexmock
+import pytest
+
+from apigentools.commands.init import InitCommand
+
+def test_init(tmpdir):
+    original_dir = os.getcwd()
+    try:
+        temp_dir = tmpdir.mkdir("test_init_git_dir")
+        os.chdir(temp_dir)
+        args = flexmock.flexmock(no_git_repo=False, projectdir=temp_dir)
+        cmd_instance = InitCommand({}, args)
+        cmd_instance.run()
+        #sets do not presume the output of os.walk() will be ordered
+        dir_entries = set(dir_entry[0] for dir_entry in os.walk("."))
+        # the layout of git repos has changed over time, this dir may or may not be present
+        if "./.git/branches" in dir_entries:
+            dir_entries.remove("./.git/branches")
+        assert dir_entries == {
```

Comment:
What do you think of using [issubset](https://docs.python.org/2/library/sets.html#set-objects) instead and only checking for the root `.git` folder? I don't think we need to test that every underlying git folder is there. And searching for exact equality will start to fail if `git` adds/removes/renames and underlying files. Not sure if thats common, but seems like we can avoid that case 😄

Reply:
I think that's a good idea -- I changed it to use `issubset`.